lbrynet/daemon/auth/auth.py

```diff
@@ -24,29 +24,11 @@ def requestAvatar(self, avatarId, mind, *interfaces):
 class PasswordChecker:
     credentialInterfaces = (credentials.IUsernamePassword,)
 
-    def __init__(self, passwords):
-        self.passwords = passwords
-
-    @classmethod
-    def load_file(cls, key_path):
-        keys = load_api_keys(key_path)
-        return cls.load(keys)
-
-    @classmethod
-    def load(cls, password_dict):
-        passwords = {key: password_dict[key].secret for key in password_dict}
-        log.info("Loaded %i api key(s)", len(passwords))
-        return cls(passwords)
+    def __init__(self, keyring: Keyring):
+        self.api_key = keyring.api_key
 
     def requestAvatarId(self, creds):
```

Comment:
Can this method name also be renamed to follow PEP standard? I realize it's not part of the PR but if it's easy.

Reply:
no, it's required to be a `ICredentialsChecker`